pillar.example

```diff
@@ -17,13 +17,7 @@ postgres:
   # Append the lines under this item to your postgresql.conf file.
   # Pay attention to indent exactly with 4 spaces for all lines.
   postgresconf: |
-    listen_addresses = '*'  # listen on all interfaces
-
-  # Backup extension for postgresql.conf file, defaults to ``.bak``.
-  # Set to False to stop creation of backup on postgresql.conf changes.
-  {%- if 'status.time' in salt.keys() %}
-  postgresconf_backup: ".backup@{{ salt['status.time']('%y-%m-%d_%H:%M:%S') }}"
-  {%- endif %}
+      listen_addresses = '*'  # listen on all interfaces
```

Comment:
Whoops, I guess I switched context too often and started to mix things :)
Indeed, YAML (and SLS) literal style scalars should be indented exactly with two spaces. Fixed, thanks!